shuup/admin/modules/orders/__init__.py

```diff
@@ -207,3 +207,27 @@ def get_menu_entries(self, request):
 
     def get_model_url(self, object, kind, shop=None):
         return derive_model_url(OrderStatus, "shuup_admin:order_status", object, kind)
+
+
+class ShipmentModule(AdminModule):
```

Comment:
why this is here?

Reply:
This is the implementation for Shippents entry button, and it's a child of Order  Admin module so I put it here. Shipment doesn't have its own module

Reply:
sorry. I pushed code to a wrong branch